LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs

```diff
@@ -86,6 +86,7 @@ public void ConfigureServices(IServiceCollection services)
                         .AddSingleton<LoRaDeviceAPIServiceBase, LoRaDeviceAPIService>()
                         .AddSingleton<WebSocketWriterRegistry<StationEui, string>>()
                         .AddSingleton<IPacketForwarder, DownstreamSender>()
+                        .AddSingleton<ClientCertificateValidatorService>()
```

Comment:
Do we want to register this unconditionally? If we don't have client certs validation enabled, we don't need it, no?